src/main/kotlin/gdx/liftoff/data/platforms/Lwjgl3.kt

```diff
@@ -211,40 +210,32 @@ jar {
   }
 }
 
-if (JavaVersion.current().isCompatibleWith(JavaVersion.VERSION_14)) {
-  tasks.jpackageImage.doNotTrackState("This task both reads from and writes to the build folder.")
-  runtime {
-    options.set(['--strip-debug',
-           '--compress', '2',
-           '--no-header-files',
-           '--no-man-pages',
-           '--strip-native-commands',
-           '--vm', 'server'])
-// you could very easily need more modules than this one.
-// use the lwjgl3:suggestModules task to see which modules may be needed.
-    modules.set([
-        'jdk.unsupported'
-    ])
-    distDir.set(file(project.layout.buildDirectory))
-    jpackage {
-      imageName = appName
-// you can set this to false if you want to build an installer, or keep it as true to build just an app.
-      skipInstaller = true
-// this may need to be set to a different path if your JAVA_HOME points to a low JDK version.
-      jpackageHome = javaHome.getOrElse("")
-      mainJar = jarName
-      if (os.contains('win')) {
-        imageOptions = ["--icon", "icons/logo.ico"]
-      } else if (os.contains('nix') || os.contains('nux') || os.contains('bsd')) {
-        imageOptions = ["--icon", "icons/logo.png"]
-      } else if (os.contains('mac')) {
-// If you are making a jpackage image on macOS, the below line should work thanks to StartupHelper.
-        imageOptions = ["--icon", "icons/logo.icns"]
-// If the above line doesn't produce a runnable executable, you can try using the below line instead of the above one.
-//        imageOptions = ["--icon", "icons/logo.icns", "--java-options", "\"-XstartOnFirstThread\""]
+construo {
+    // name of the executable
+    name.set(appName)
+    // human-readable name, used for example in the `.app` name for macOS
+    humanName.set(appName)
+    // Optional, defaults to project version
+    version.set("0.0.0")
+
+    targets.configure {
+      create("linuxX64", Target.Linux) {
+        architecture.set(Target.Architecture.X86_64)
+        jdkUrl.set("https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.11%2B9/OpenJDK17U-jdk_x64_linux_hotspot_17.0.11_9.tar.gz")
+      }
+      create("macM1", Target.MacOs) {
```

Comment:
Is it possible to target x64 MacOS as well? I know we have people using x64 Macs in the libGDX Discord part of the community, and there's probably more outside of it. If it's going to be a major hassle, don't worry about it; new Macs are all going to be AARCH64 anyway.

Reply:
[eae40ea](https://github.com/libgdx/gdx-liftoff/pull/155/commits/eae40eaaebed2971389f9fe3054f1b337f36097b) yep! Quite easy too